kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -212,36 +236,52 @@ def run(self, labels: list, analysis_id=None):
                 return
 
         if analysis_id is None:
-            with cloned_repo(self, depth=1) as repo:
+            with cloned_repo(self, self.project.default_branch, depth=1) as repo:
                 self.repo = repo
-                if not os.path.isfile("Pipfile"):
-                    _LOGGER.warning("Pipfile not found in repo... Creating issue")
-                    issue = self.get_issue_by_title("Missing Pipfile")
-                    if not issue:
-                        self.project.create_issue(
-                            title="Missing Pipfile",
-                            body="Check your repository to make sure Pipfile exists",
-                            labels=labels,
-                        )
-
-                    return False
 
-                lib.advise_here(
-                    nowait=True,
-                    origin=(f"{self.service_url}/{self.slug}"),
-                    source_type=ThothAdviserIntegrationEnum.KEBECHET,
-                    kebechet_metadata=self.metadata,
-                )
+                with open(".thoth.yaml", "r") as f:
+                    thoth_config = yaml.safe_load(f)
+
+                if self.runtime_environment:
+                    if self.runtime_environment not in [
+                        e["name"] for e in thoth_config["runtime_environments"]
+                    ]:
+                        # This is not a warning as it is expected when users remove and change runtime_environments
+                        _LOGGER.info("Requested runtime does not exist in target repo.")
```

Comment:
This might be rare case on the user side. On the other hand, if we supply bad runtime environment name no message will be reported in monitoring systems. Might be good idea to consider this as an error:

```suggestion
                        _LOGGER.error("Requested runtime does not exist in target repo.")
```